app/uk/gov/hmrc/valuationofficeagencycontactfrontend/connectors/LightweightContactEventsConnector.scala

```diff
@@ -56,7 +52,9 @@ class LightweightContactEventsConnector @Inject()(http: HttpClient,
       .map {
         response =>
           response.status match {
-            case 200 => Success(200)
+            case 200 =>
+              AuditingService.sendEvent("sendenquirytoVOA", json)
+              Success(200)
             case status => {
               Logger.warn("Received status of " + status + " from upstream service")
               Failure(new RuntimeException("Received status of " + status + " from upstream service"))
```

Comment:
so we have already got exceptions for these sort of errors. if you do case status if status >= 400 < 500 Upstream4XXException and then case status if status >= 500 then Upstream5XXException

Reply:
That was as I inherited the project. I don't want to change any more that I have to at this stage. Can I leave this for now?